std/internal/mmm.d

```diff
@@ -0,0 +1,337 @@
+/**
+This module provides OOM-checked and memory safe interfaces to C's $(D malloc)
+and $(D calloc) functions.
+
+Acronyms:
+$(B OOM:) $(I Out Of Memory), the condition in which an allocator has no more
+memory space to offer.
+
+Copyright: Jakob Øvrum 2015
+
+License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: Jakob Øvrum
+*/
+module std.internal.mmm;
```

Comment:
For posterity, it was meant to be "Manual Memory Management". It's an internal module so I didn't put much thought into it